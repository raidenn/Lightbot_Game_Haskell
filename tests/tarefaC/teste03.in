abcdd
aacAa
Aaaaa
0 0 E
LAAAEAL
