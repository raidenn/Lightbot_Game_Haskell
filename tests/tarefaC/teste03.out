0 0
3 1
FIM 7
